### PR TITLE
[Port] Bunch of QoL, new stuff and fixes from Azure Peak

### PR DESCRIPTION
**Pull request by Vladmar11**
1. Fixed bug metal heat with tongs.

2. Possibility to add material from tongs to recipe on anvil. https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/381

3. Added confirm for tabards, shields. "You get a prompt to confirm whether you want to keep your shield heraldry. You also have to right click shields to set their heraldry now." - EvenInDeathIStillServe https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/100

4. Added Preload Ambience feature https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/331

5. Added bold text for weapons when examining characters https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/75

6. Added wall clock functionality (show the time and day) https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/101

7. Ported one-click collection of arrows and bolts into quiver. https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/327

8. Added sounds of weapon draw and stow https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/349

9. Added the ability to drink from a fountain https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/350

10. Increase fuel for greater furnace. "You can fill up smelters with fuel up to 30 minutes. One coal is 10 minutes." - EvenInDeathIStillServe https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/370

11. Fix armour damage (upper armour takes damage, then lower armour) https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/326

12. Ported autosmithing, it's less miserable for smiths players. "Striking an ingot will begin to automatically continue striking, until you either fail a strike, the bar cools, or the item finishes. You can still manually strike. Auto smithing take 4 times as many strikes as manual smithing." - EvenInDeathIStillServe https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/294

Feat. SayonaraEcho

image
image
image
image

### CODE REVIEW

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -225,3 +225,34 @@
...
+	'sound/music/area/manorgarri.ogg',
+	'sound/music/area/sargoth.ogg',
+	'sound/music/area/septimus.ogg',
+	'sound/music/area/sewers.ogg',
+	'sound/music/area/shop.ogg',
+	'sound/music/area/spidercave.ogg',
+	'sound/music/area/towngen.ogg',
+	'sound/music/area/townstreets.ogg',
+	'sound/music/jukeboxes/tav3.ogg'
+	))
```
Comment (Gyrandola) [inline code/modules/admin/verbs/playsound.dm:250]: are these actually used? azure peek has a different map with potentially different areas.

**Comment by Gyrandola** [comment]
Also, unless I missed it, preloading should probably be done on spawn so players dont have to bother clicking the button

File: code/modules/admin/verbs/playsound.dm (hunk 1 above)
Comment (Vladmar11) [reply code/modules/admin/verbs/playsound.dm]: Fixed it. Now only the sounds that are used.

**Comment by Vladmar11** [comment]
> Also, unless I missed it, preloading should probably be done on spawn so players dont have to bother clicking the button

No idea yet how to do it without breaking anything.

**Review by Gyrandola — APPROVED** [review]

**Comment by wow50000** [comment]
This PR made smithing use way more stamina than it used to

**Pull request merged**